packit/config/aliases.py

```diff
@@ -20,6 +20,10 @@
     "fedora-latest-stable": ["fedora-37"],
     "fedora-branched": ["fedora-36", "fedora-37", "fedora-38"],
     "epel-all": ["epel-7", "epel-8", "epel-9"],
+    "opensuse-leap-all": (
+        leap_all := ["opensuse-leap-15.5", "opensuse-leap-15.4", "opensuse-leap-15.3"]
```

Comment:
Is the minor version necessary in opensuse?

Reply:
Yes it is, they are different versions co-exist in parallel.